rules/github-mobile/rule.md

```diff
@@ -0,0 +1,45 @@
+---
+seoDescription: Discover how having GitHub on your phone helps you stay connected to your projects and unblock your team from anywhere.
+type: rule
+title: Do you have GitHub on your phone?
+uri: github-mobile
+authors:
+  - title: Seth Daily
+    url: https://ssw.com.au/people/seth-daily
+related:
+  - pomodoro
+  - github-notifications
+created: 2024-10-05T13:42:55.753Z
+guid: aed50ddb-6105-491e-9963-5cc514766dc9
+---
+
+As a developer you're likely using GitHub all the time - it's one of the main repositories for code. When you're keeping a good work-life balance, it's not ideal for someone to be blocked by a pull request review that would take you 1 minute - just because you're out getting some fresh air.
+
+<!--endintro-->
+
+Having GitHub on your mobile helpw you unblock people quickly, and leave comments all while you’re on the go. Devs are more productive [when they get regular breaks](https://www.ssw.com.au/rules/pomodoro) - so why not review pull requests in the park!
+
+### Why You Should Have GitHub on Your Phone
+
+* **Unblock others** - Nothing is more frustrating for a developer than waiting hours or even days for a small PR approval or a comment on an issue
+
+* **Stay updated** - It’s easier to fall behind on comments and issues if you’re only able to check GitHub from your computer.
+
+* **[Get notifications](https://www.ssw.com.au/rules/github-notifications/)**  GitHub's mobile app lets you control which notifications you receive
```

Comment:
Hey @sethdaily 

- [ ] 1. Missing the dash after the prefix
- [ ] 2. Make the link relative...